server/src/main/java/org/apache/iotdb/db/query/reader/universal/PriorityMergeReader.java

```diff
@@ -42,6 +47,11 @@
   public PriorityMergeReader() {
   }
 
+  public PriorityMergeReader(int floatPrecision) {
```

Comment:
If we use a List<ReadOnlyMemchunk> , this class should not be changed.
is this a bug before?

Reply:
It's for formatting data in memory